adapters/platform_linux.c

```diff
@@ -31,11 +31,13 @@ int platform_init(void)
 {
     int result = 0;
 #ifndef DONT_USE_UPLOADTOBLOB
+#if !defined(HTTP_DISABLED)
```

Comment:
Should the include for `#include "azure_c_shared_utility/httpapiex.h"` be conditional as well?

Reply:
Fixed!